plugins/tiddlywiki/tiddlyweb-sse/sse-client.js

```diff
@@ -0,0 +1,73 @@
+/*\
+title: $:/plugins/tiddlywiki/tiddlyweb-sse/sse-client.js
+type: application/javascript
+module-type: startup
+
+Miscellaneous startup logic for both the client and server.
+
+\*/
+(function() {
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+// Export name and synchronous status
+exports.name = "hook-tiddlyweb-sse";
+exports.after = ["startup"];
+exports.platforms = ["browser"];
+exports.synchronous = true;
+exports.startup = function() {
+	var source = null;
+	$tw.hooks.addHook("th-syncer-status-response",function(syncer,success) {
+		// technically any syncer instance can be used, but we only use the global one for this
+		if(!success || syncer.syncadaptor.name !== "tiddlyweb" || $tw.syncer !== syncer) { return; }
+		// check if we have a previous one and close it if we do
+		if(source && source.readyState !== source.CLOSED) { source.close(); }
+		// Get the mount point in case a path prefix is used
+		var host = syncer.syncadaptor.getHost();
+		// Make sure it ends with a slash (it usually does)
+		if(host[host.length - 1] !== "/") { host += "/"; }
+		var endpoint = host + "events/plugins/tiddlywiki/tiddlyweb-sse/wiki-change";
```

Comment:
IMO it should be `events/plugins/tiddlywiki/tiddlyweb-sse/wiki-changed` see the "d" at the end